network/glewlwyd/glewlwyd.SlackBuild

```diff
@@ -85,6 +85,7 @@ cd build
     -DCMAKE_INSTALL_PREFIX=/usr \
     -DCMAKE_INSTALL_DOCDIR=doc/$PRGNAM-$VERSION \
     -DCMAKE_INSTALL_SYSCONFDIR=/etc \
+    -DCMAKE_INSTALL_MANDIR=/usr/man \
```

Comment:
Seems this does not work as expected for whatever reason. The man pages still end up in /usr/share/man

Reply:
I've finished fully rebuilding all glewlwyd and requirements, on a clean 15.0 almost-fully-patched chroot.
I have no man page in usr/share/man, they are all in usr/man, glewlwyd's here : `usr/man/man8/glewlwyd.8`
And without the above line, they were effectively in `usr/share/man/man8/glewlwyd.8`
So, if it really isn't working for you, please, try and tell me what kind of environment, or tool, you're using ?

Reply:
Apologies, I ran explodepkg on the previous package again instead of the newly rebuilt one. This is staged to my branch now.